### PR TITLE
Top ranking

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Add elements html for top ranking
- Add styles for top ranking

### CODE REVIEW

File: assets/scss/templates/_reportes.scss [hunk 1]
```diff
@@ -171,3 +171,46 @@
     }
   }
 }
+
+/* Top ranking */
+.wrapper-ranking {
+  max-width: $dp-wrapper-max-width;
+  margin: 25px auto;
```
Comment (IgnacioRodrigues) [inline assets/scss/templates/_reportes.scss]: Maybe use variables ?

File: assets/scss/templates/_reportes.scss [hunk 2]
```diff
@@ -171,3 +171,46 @@
...
+    font-size: $dp-font-size-small;
+    color: $dp-color-darkgrey;
+    line-height: 18px;
+  }
+  .page-ranking--item {
+    border-top: 1px solid $dp-color-sylver;
+    border-bottom: 1px solid $dp-color-sylver;
+    padding: 42px 30px;
+    background: #f6f6f6;
+    margin-bottom: 5px;
```
Comment (IgnacioRodrigues) [inline assets/scss/templates/_reportes.scss]: In this case as well?

File: assets/scss/templates/_reportes.scss [hunk 3]
```diff
@@ -79,7 +79,7 @@
 }
 
 /* Section KPI */
-.container-kpi {
+.container-reports {
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: It is a breaking change, what do you think about keeping the old class (with a `// TODO` comment), in order to keep compatibility?

This kind of practices is not required right now, but we should at least think about it in order to learn to think in that way.

File: assets/scss/templates/_reportes.scss [hunk 4]
```diff
@@ -171,3 +171,46 @@
...
+  margin: 25px auto;
+  font-family: $dp-font-family;
+
+  .reports-box {
+    @include dp-box($dp-color-sylver, 100%, 100%, 100%);
+    position: relative;
+    padding: 0;
+  }
+  .title-ranking {
+    padding: 40px 30px;
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Should it not using any space variable?

File: assets/scss/templates/_reportes.scss (hunk 1 above)
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Should it not use any space variable?

File: assets/scss/templates/_reportes.scss [hunk 5]
```diff
@@ -171,3 +171,46 @@
...
+    display: inline-block;
+    font-size: $dp-font-size-small;
+    color: $dp-color-darkgrey;
+    line-height: 18px;
+  }
+  .page-ranking--item {
+    border-top: 1px solid $dp-color-sylver;
+    border-bottom: 1px solid $dp-color-sylver;
+    padding: 42px 30px;
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Should it not use any space variable?

File: assets/scss/templates/_reportes.scss [hunk 6]
```diff
@@ -171,3 +171,46 @@
...
+    background: #f6f6f6;
+    margin-bottom: 5px;
+    min-height: 208px;
+
+    .text-ranking {
+      margin: 16px 0;
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Should it not use any space variable?

(Maybe there are more)

File: assets/templates/reports-filters.html [hunk 7]
```diff
@@ -120,6 +120,21 @@ <h6 class="subtitle-kpi">contactos nuevos</h6>
         </ul>
       </div><!-- reports-box -->
     </div>
+    <div class="wrapper-ranking">
+      <div class="reports-box">
+        <small class="title-ranking">TOP PAGINAS MAS VISITADAS</small>
```
Comment (andresmoschini) [inline assets/templates/reports-filters.html]: Could you avoid uppercase text and use styles in cases like this?

File: assets/scss/templates/_reportes.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/scss/templates/_reportes.scss (hunk 2 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/scss/templates/_reportes.scss (hunk 3 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Lo cambie por que es un container general para "reportes" no solo de kpi. Pero como quieran, se cambia

File: assets/scss/templates/_reportes.scss (hunk 4 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/scss/templates/_reportes.scss (hunk 1 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/scss/templates/_reportes.scss (hunk 5 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/scss/templates/_reportes.scss (hunk 6 above)
Comment (GusBaamonde) [reply assets/scss/templates/_reportes.scss]: Ok sorry, forget me. I correct

File: assets/templates/reports-filters.html (hunk 7 above)
Comment (GusBaamonde) [reply assets/templates/reports-filters.html]: Ok, I correct

File: assets/scss/templates/_reportes.scss [hunk 8]
```diff
@@ -171,3 +171,47 @@
...
+/* Top ranking */
+.wrapper-ranking {
+  max-width: $dp-wrapper-max-width;
+  margin: $dp-spaces--lv4 auto;
+  font-family: $dp-font-family;
+
+  .reports-box {
+    @include dp-box($dp-color-sylver, 100%, 100%, 100%);
+    position: relative;
+    padding: 0;
```
Comment (andresmoschini) [inline assets/scss/templates/_reportes.scss]: Should you use a variable here?

**Pull request merged**